### PR TITLE
upgraded to 7.9.0.Final

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: _config/pom.yml [hunk 1]
```diff
@@ -14,55 +14,55 @@ kieExecutionServerDescription: Standalone execution server that can be used to r
...
 
-    documentationHtmlSingle: https://docs.jboss.org/drools/release/7.8.0.Final/drools-docs/html_single/index.html
+    documentationHtmlSingle: https://docs.jboss.org/drools/release/7.9.0.Final/drools-docs/html_single/index.html
 
-    KIE_API_documentationJavadoc: https://docs.jboss.org/drools/release/7.8.0.Final/kie-api-javadoc/index.html
+    KIE_API_documentationJavadoc: https://docs.jboss.org/drools/release/7.9.0.Final/kie-api-javadoc/index.html
 
-    droolsWhatsNew: https://docs.jboss.org/drools/release/7.8.0.Final/drools-docs/html_single/#_droolsreleasenoteschapter
+    droolsWhatsNew: https://docs.jboss.org/drools/release/7.9.0.Final/drools-docs/html_single/#_droolsreleasenoteschapter
     droolsReleaseNotes: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12337237
```
Comment (jomarko) [inline _config/pom.yml]: I think we should update from:
```
droolsReleaseNotes: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12337237
```
to
```
droolsReleaseNotes: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12338258
```

File: _config/pom.yml [hunk 2]
```diff
@@ -14,55 +14,55 @@ kieExecutionServerDescription: Standalone execution server that can be used to r
...
```
Comment (jomarko) [inline _config/pom.yml]: I think we should update from:
```
droolsReleaseNotes: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12337237
```
to
```
droolsReleaseNotes: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12338258
```

File: _config/pom.yml (hunk 1 above)
Comment (mbiarnes) [reply _config/pom.yml]: @jomarko DONE

File: _config/pom.yml (hunk 2 above)
Comment (mbiarnes) [reply _config/pom.yml]: @jomarko DONE

**Comment by mbiarnes** [comment]
@jomarko @mariofusco if anyone of you guys approves this I will merge it

File: _config/pom.yml [hunk 3]
```diff
@@ -96,7 +96,7 @@ latest6:
     droolsExpert_documentationHtml: https://docs.jboss.org/drools/release/6.5.0.Final/drools-expert-docs/html/index.html
     droolsExpert_documentationPdf: https://docs.jboss.org/drools/release/6.5.0.Final/drools-expert-docs/pdf/drools-expert-docs.pdf
 
-    droolsReleaseNotes: https://docs.jboss.org/drools/release/6.5.0.Final/drools-docs/html_single/#DroolsReleaseNotesChapter
+    droolsReleaseNotes: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12338258
```
Comment (jomarko) [inline _config/pom.yml]: This is section for 6.5.x, no change needed.

**Pull request merged**